mitodl/edx-api-client
# feat: validate user registration api

### What are the relevant tickets?
[#4211](https://github.com/mitodl/hq/issues/4211)

### Description (What does it do?)
This PR adds a new API to validate user information during registration or profile updates.
Related PR: https://github.com/mitodl/mitxpro/pull/2994

### Screenshots (if appropriate):


### How can this be tested?
- Fetch, check out to this branch, and install edx-api-client with editable mode.
- See the testing instruction mentioned in this PR  https://github.com/mitodl/mitxpro/pull/2994 and test validate_user_registration API endpoint.

### Additional Context
